forms-flow-web/src/components/ServiceFlow/list/sort/CreateNewFilter.js

```diff
@@ -512,6 +516,7 @@ export default function CreateNewFilterDrawer({
   };
 
   const toggleDrawer = () => {
+    dispatch(fetchAllBpmProcesses());
```

Comment:
call fetchAll bpm process on when the create new filter mount 
`
  useEffect(() => {
    if (openFilterDrawer) {
   dispatch(fetchAllBpmProcesses());
    }
  }, [openFilterDrawer]);
`

Reply:
good catch. Thanks